src/github/pullRequestManager.ts

```diff
@@ -687,13 +740,6 @@ export class PullRequestManager implements vscode.Disposable {
 		};
 	}
 
-	public mayHaveMorePages(): boolean {

```

Comment:
Unused and not really fundamentally compatible with the fact that there are many different queries that may or may not have more pages.